tool/nixvirt.py

```diff
@@ -121,7 +121,7 @@ def _descriptionXMLText(self,lvobj):
         # VIR_NETWORK_XML_INACTIVE
         return lvobj.XMLDesc(flags=1)
     def _getDependents(self,obj):
-        networknames = [str(name) for name in obj.descriptionXMLETree().xpath("/network/name")]
+        networknames = [str(name.text) for name in obj.descriptionXMLETree().xpath("/network/name")]
```

Comment:
This doesn't need `str()`, does it?

Reply:
I'm OK with using `str()` for the other ones, because I'd prefer to convert "smart strings" to strings before comparison.

Reply:
Actually you're right - no need to do the conversion, from what I can see [in the docs](https://lxml.de/api/lxml.etree._Element-class.html) this is just a regular string!

Reply:
I've made the change but haven't tested on my machine yet - will do that within the next hour just to double check the docs haven't lied...